### PR TITLE
simplify gh pages episode

### PR DESCRIPTION
**Pull request by bast**
This is based on https://coderefinery.github.io/github-without-command-line/05-websites/.

The significant change is that instead of providing step-by-step and proving a template to start with, we point to the official documentation which is IMO really clear.

Advantage (in my eyes):
- we have less to maintain on our side
- their docs offer more flexibility: some learners will prefer testing setting up a homepage, some will prefer setting up a project page

### CODE REVIEW

File: _episodes/06-gh-pages.md [hunk 1]
```diff
@@ -1,92 +1,49 @@
 ---
 layout: episode
-title: "Deploying a project website or homepage to GitHub Pages"
-teaching: 0
-exercises: 20
+title: Hosting websites/homepages on GitHub Pages
+teaching: 20
+exercises: 0
```
Comment (rkdarst) [inline _episodes/06-gh-pages.md:5]: It seems like it's mostly exercise - is 20/0 what it should be?

File: _episodes/06-gh-pages.md [hunk 2]
```diff
@@ -1,92 +1,49 @@
...
-> - First import [https://github.com/coderefinery/gh-pages-example](https://github.com/coderefinery/gh-pages-example) and clone the imported repository.
-> ```shell
-> $ git clone https://github.com/user/gh-pages-example.git  # adapt user
-> $ cd gh-pages-example
-> ```
-> - Check that you are indeed on the `gh-pages` branch.
-> ```shell
-> $ git status
-> ```
-> - Make some changes to `index.md` or `about.md` or `team.md`, commit and push.
```
Comment (rkdarst) [inline _episodes/06-gh-pages.md:54]: I agree with using the upstream pages.github.com reference.  But, can we make it more specific what a possible action should be?  As in, "make a markdown index.md and add a new page team.md".  My worry is that being too open-ended means some people don't know where to start (and those that know, can do what they want anyway).

File: _episodes/06-gh-pages.md (hunk 1 above)
Comment (bast) [reply _episodes/06-gh-pages.md:5]: Oops, yes, I meant the exact opposite. I will adjust.

File: _episodes/06-gh-pages.md (hunk 2 above)
Comment (bast) [reply _episodes/06-gh-pages.md:54]: Good suggestion. I will tomorrow go through the official step-by-step just to see what I get precisely and adjust.

Comment (bast) [reply _episodes/06-gh-pages.md:54]: Done.

**Comment by bast** [comment]
Another motivation why I wanted this to be short:
- we never manage this one anyway with time
- if it is short, my hope is that participants look at it and hopefully get curious

**Comment by bast** [comment]
Good idea, I will add that comparison.

**Pull request merged**